docs/api/selector.md

```diff
@@ -12,6 +12,16 @@ follows:
 - class syntax (`.foo`, `.foo-bar`, etc.)
 - tag syntax (`input`, `div`, `span`, etc.)
 - id syntax (`#foo`, `#foo-bar`, etc.)
+- attr syntax(`[foo="bar"]`, `[foo]`, etc.);
+
+A note on attribute selectors: For compatibilty, you can use either the
+react prop name, or the native html selector. For example, the following
```

Comment:
I started on an implementation to use seperate syntax (`{}` for props, `[]` for html attr). It seems feasible. In shallow renders, I'm basically faking it by mapping the html attr selector to the matching react prop since I don't get an actual render. I'm just waiting on word from @lelandrichardson if that is the way we want this togo.